tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "exactOptionalPropertyTypes": true,
     "forceConsistentCasingInFileNames": true,
     "isolatedModules": true,
-    "module": "es2022",
+    "module": "node16",
```

Comment:
The latest version of TypeScript complained that this field did not match the value in the `moduleResolution` field, so I changed it.